xbmc/utils/StringUtils.cpp

```diff
@@ -756,6 +756,25 @@ std::vector<std::string> StringUtils::Split(const std::string& input, const char
   return results;
 }
 
+std::vector<std::string> StringUtils::Split(const std::string& input, const std::vector<std::string> &delimiters)
```

Comment:
Yes I considered it, and even tested it.
SplitMulti specifically has order/prescidence to the delimiters, it does something more than just treat all delimiters the same. Because of that it is slower than simply applying all the delimiters equally which is what this does along with the other single delimiter variants of Split.